### PR TITLE
Hand on table addendum

### PR DESCRIPTION
**Pull request by andypryor**
Allows spiker to place hand on playing surface while completing a lob.

If we get more 👍  than 👎  by 8/17 we'll merge.

### CODE REVIEW

File: rules.txt [hunk 1]
```diff
@@ -5,21 +5,27 @@
...
     2.1.1 The first player to strike the ball on a return is the "setter".  The partner of the setter when finishing the return is the "spiker".  The setter may choose to return the ball directly,  or they may strike the ball so that the "spiker" is able to return the ball to the opponents side of the table.  When the "setter" and "spiker" both hit the ball on a return this is known as a "lob".
-    2.1.2  Lob Scoring
-      2.1.2.1  A lob that scores immediately after a 1 hit return shall be worth 2 points.
-      2.1.2.2  A lob that scores after a lob from the opponents shall be worth (N+1) points,  where N is the number of consective lobs that have occurred between both teams since a 1 hit return. For example, if the serving team hits a lob at the first legal opportunity, the receiving team fields and converts their own lob, it shall be worth 3 points. 
+    2.1.2  In the process of completing a lob,  it is permissible for one hand of the spiker to come in contact with any part of the table.
```
Comment (bclymer) [inline rules.txt:8]: giphy
^ Andy trying to fix his broken game.

**Comment by DDropinski** [comment]
I really don't have an opinion on this, honestly. I'd be fine either way, it should just be clear in the rules and publicized.

**Comment by DDropinski** [comment]
If I had to vote, I'd probably say 👍 as a default.

**Comment by andypryor** [comment]
As a repo admin,  you have to vote @DDropinski

**Comment by ericwclymer** [comment]
Love it. :+1: :shipit:

**Comment by afilippi** [comment]
You've got all the 👍  you really need. :shipit:

**Pull request merged**